### ISSUE TITLE
[Edit Last.fm Scrobbles] Bug - Can't save with accented character in edit field

### ISSUE THREAD

**Issue by hujiba** [CennoxX/userscripts#14]
When I try to edit the artist of a track to anything with an accented character (é, á, etc.), nothing happens after clicking the save button. In the console, this message is shown:
```Error from Last.fm: <?xml version="1.0" encoding="UTF-8"?>
<lfm status="failed">
  <error code="13">Invalid method signature supplied</error>
</lfm>```

**Comment by hujiba** [comment]
Not 100% sure, but based on my limited JavaScript knowledge and a little bit of messing around with the code, I think the problem is in the `lfmd5` function. No idea what the problem is though :)

**Comment by CennoxX** [comment]
This worked before the fix for #12, do you have any ideas about that, @nicoleahmed?
@hujiba For the moment you could use the old version https://greasyfork.org/de/scripts/485278-edit-last-fm-scrobbles?version=1372518
